### PR TITLE
feat(bundler): hide output from shell scripts unless --verbose is passed (fixes #888)

### LINKED ISSUE #888: Clean up .appimage build script output
**Is your feature request related to a problem? Please describe.**
The `.deb` build script just outputs a nice `bundling "[FILE]"`, while the `.appimage` build script outputs this mess:
```
info: running build_appimage.sh
+ export ARCH=x86_64
+ ARCH=x86_64
+ mkdir -p theia.AppDir
+ cp -r ../deb/theia_0.1.0_amd64/data/usr theia.AppDir
+ cd theia.AppDir
+ wget -q -O AppRun https://github.com/AppImage/AppImageKit/releases/download/continuous/AppRun-x86_64
+ chmod +x AppRun
+ cp usr/share/icons/hicolor/256x256/apps/theia.png theia.png
+ cd ..
+ echo '[Desktop Entry]'
+ echo Version=1.0
+ echo 'Comment=A Tauri App'
+ echo Exec=theia
+ echo Icon=theia
+ echo Name=THEIA
+ echo Terminal=false
+ echo Type=Application
+ echo 'Categories=X-Web;'
+ cp theia.desktop theia.AppDir/usr/share/applications/theia.desktop
+ mv theia.desktop theia.AppDir/theia.desktop
+ mksquashfs theia.AppDir theia.squashfs -root-owned -noappend
Parallel mksquashfs: Using 16 processors
Creating 4.0 filesystem on theia.squashfs, block size 131072.
[=============================================================\] 1598/1598 100%

Exportable Squashfs 4.0 filesystem, gzip compressed, data block size 131072
	compressed data, compressed metadata, compressed fragments,
	compressed xattrs, compressed ids
	duplicates are removed
Filesystem size 63590.30 Kbytes (62.10 Mbytes)
	31.39% of uncompressed filesystem size (202561.36 Kbytes)
Inode table size 4660 bytes (4.55 Kbytes)
	59.94% of uncompressed inode table size (7774 bytes)
Directory table size 425 bytes (0.42 Kbytes)
	39.87% of uncompressed directory table size (1066 bytes)
Number of duplicate files found 3
Number of inodes 45
Number of files 18
Number of fragments 1
Number of symbolic links  0
Number of device nodes 0
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 27
Number of ids (unique uids + gids) 1
Number of uids 1
	root (0)
Number of gids 1
	root (0)
+ wget -q -O appimagetool https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
+ chmod +x appimagetool
+ lsmod
+ grep -q fuse
+ ./appimagetool --appimage-extract
squashfs-root/.DirIcon
squashfs-root/AppRun
squashfs-root/appimagetool.desktop
squashfs-root/appimagetool.png
squashfs-root/usr
squashfs-root/usr/bin
squashfs-root/usr/bin/AppRun
squashfs-root/usr/bin/appimagetool
squashfs-root/usr/bin/desktop-file-validate
squashfs-root/usr/bin/zsyncmake
squashfs-root/usr/lib
squashfs-root/usr/lib/appimagekit
squashfs-root/usr/lib/appimagekit/mksquashfs
squashfs-root/usr/lib/libffi.so.5
squashfs-root/usr/lib/libgio-2.0.so
squashfs-root/usr/lib/libgio-2.0.so.0
squashfs-root/usr/lib/libgio-2.0.so.0.5600.0
squashfs-root/usr/lib/libglib-2.0.so
squashfs-root/usr/lib/libglib-2.0.so.0
squashfs-root/usr/lib/libglib-2.0.so.0.5600.0
squashfs-root/usr/lib/libgmodule-2.0.so
squashfs-root/usr/lib/libgmodule-2.0.so.0
squashfs-root/usr/lib/libgmodule-2.0.so.0.5600.0
squashfs-root/usr/lib/libgobject-2.0.so
squashfs-root/usr/lib/libgobject-2.0.so.0
squashfs-root/usr/lib/libgobject-2.0.so.0.5600.0
squashfs-root/usr/lib/libgthread-2.0.so
squashfs-root/usr/lib/libgthread-2.0.so.0
squashfs-root/usr/lib/libgthread-2.0.so.0.5600.0
squashfs-root/usr/lib/libpcre.so
squashfs-root/usr/lib/libpcre.so.1
squashfs-root/usr/lib/libpcre.so.1.2.11
squashfs-root/usr/lib/libpcrecpp.so
squashfs-root/usr/lib/libpcrecpp.so.0
squashfs-root/usr/lib/libpcrecpp.so.0.0.1
squashfs-root/usr/lib/libpcreposix.so
squashfs-root/usr/lib/libpcreposix.so.0
squashfs-root/usr/lib/libpcreposix.so.0.0.6
squashfs-root/usr/share
squashfs-root/usr/share/applications
squashfs-root/usr/share/applications/appimagetool.desktop
squashfs-root/usr/share/icons
squashfs-root/usr/share/icons/hicolor
squashfs-root/usr/share/icons/hicolor/128x128
squashfs-root/usr/share/icons/hicolor/128x128/apps
squashfs-root/usr/share/icons/hicolor/128x128/apps/appimagetool.png
squashfs-root/usr/share/metainfo
squashfs-root/usr/share/metainfo/appimagetool.appdata.xml
+ ./squashfs-root/AppRun theia.AppDir theia.AppImage
appimagetool, continuous build (commit 0880085), build 2133 built on 2020-07-09 12:25:52 UTC
Using architecture x86_64
Deleting pre-existing .DirIcon
Creating .DirIcon symlink based on information from desktop file
WARNING: AppStream upstream metadata is missing, please consider creating it
         in usr/share/metainfo/theia.appdata.xml
         Please see https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps
         for more information or use the generator at http://output.jsbin.com/qoqukof.
Generating squashfs...
Parallel mksquashfs: Using 16 processors
Creating 4.0 filesystem on theia.AppImage, block size 131072.
[=============================================================\] 1598/1598 100%

Exportable Squashfs 4.0 filesystem, gzip compressed, data block size 131072
	compressed data, compressed metadata, compressed fragments,
	compressed xattrs, compressed ids
	duplicates are removed
Filesystem size 63590.34 Kbytes (62.10 Mbytes)
	31.39% of uncompressed filesystem size (202561.42 Kbytes)
Inode table size 4686 bytes (4.58 Kbytes)
	60.02% of uncompressed inode table size (7807 bytes)
Directory table size 436 bytes (0.43 Kbytes)
	40.30% of uncompressed directory table size (1082 bytes)
Number of duplicate files found 3
Number of inodes 46
Number of files 18
Number of fragments 1
Number of symbolic links  1
Number of device nodes 0
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 27
Number of ids (unique uids + gids) 1
Number of uids 1
	root (0)
Number of gids 1
	root (0)
Embedding ELF...
Marking the AppImage as executable...
Embedding MD5 digest
Success

Please consider submitting your AppImage to AppImageHub, the crowd-sourced
central directory of available AppImages, by opening a pull request
at https://github.com/AppImage/appimage.github.io
/home/noah/Documents/html/theia2/examples/tauri/src-tauri/target/release/bundle/appimage/theia.AppDir should be packaged as theia.AppImage
+ rm -rf ./squashfs-root`
```

**Describe the solution you'd like**
IMO, the `.appimage` build script should have the same output as the `.deb` as to not clutter terminal output. The current output from it feels really out of place and TMI.

### PR DESCRIPTION
**Pull request by nklayman**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

As per #888, I think the platform build scripts' output is way too verbose. This PR will hide all the output from those scripts so that they match the behavior of pure rust bundlers.

## To Do:

- [x] Provide a configuration to show the script's output
- [x] Show output on error (`stderr` always shows some info text for some reason, even on success)

Rather than fix stderr, I just added a notice about `--verbose` if the script fails (dmg/appimage only)

### CODE REVIEW

File: cli/tauri-bundler/src/bundle/appimage_bundle.rs [hunk 1]
```diff
@@ -74,7 +74,7 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<PathBuf>> {
 
   // create the shell script file in the target/ folder.
   let sh_file = output_path.join("build_appimage.sh");
-  common::print_bundling(format!("{:?}", &appimage_path).as_str())?;
```
Comment (lucasfernog) [inline cli/tauri-bundler/src/bundle/appimage_bundle.rs:77]: why did you change this from the full path to the filename? Every other place is still the full path.

Comment (nklayman) [reply cli/tauri-bundler/src/bundle/appimage_bundle.rs:77]: The `.deb` bundle log shows just the file name. The full path is displayed after anyways.

**Review by lucasfernog — APPROVED** [review]
